src/utils/beneficial.owner.individual.ts

```diff
@@ -1,8 +1,18 @@
 import { NextFunction, Request, Response } from "express";
 import { Session } from "@companieshouse/node-session-handler";
+import { addCeasedDateToTemplateOptions } from "../utils/update/ceased_date_util";
+import { saveAndContinue } from "../utils/save.and.continue";
+import { logger } from "../utils/logger";
+import { v4 as uuidv4 } from "uuid";
```

Comment:
it is just a small detail, but is it possible to keep the library imports at the top and then the local imports?

What do you think?

Reply:
agreed. resolved now.